workspace-manager-clienttests/tools/render-config.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
I think this file should live in the clienttests/ directory rather than a tools/ subdirectory. I don't think we expect there to be so many supporting scripts / tools that they'll make the parent directory too full, and it's more confusing to have it separated from the build file.

Reply:
Moved the `render-config.sh` file one level up.